tests/UnitTests/CoreGraphics.drawing/DrawingTest.cpp

```diff
@@ -0,0 +1,146 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#include "DrawingTest.h"
+
+#include <CoreFoundation/CoreFoundation.h>
+#include <ImageIO/ImageIO.h>
+
+#include <Starboard/SmartTypes.h>
+#include <memory>
+
+static const CGSize g_defaultCanvasSize{ 512.f, 256.f };
+
+woc::unique_cf<CGColorSpaceRef> testing::DrawTest::s_deviceColorSpace;
+
+void testing::DrawTest::SetUpTestCase() {
+    s_deviceColorSpace.reset(CGColorSpaceCreateDeviceRGB());
+}
+
+void testing::DrawTest::TearDownTestCase() {
+    s_deviceColorSpace.release();
+}
+
+CGSize testing::DrawTest::CanvasSize() {
```

Comment:
Actually, the UIKitMimicTest fixture does exactly that; It presents a 2x-scaled context.